Foundation/NSURLSession/NSURLSessionTask.swift

```diff
@@ -560,10 +556,8 @@ fileprivate extension URLSessionTask {
         easyHandle.set(followLocation: false)
         easyHandle.set(customHeaders: curlHeaders(for: request))
 
-        //Options unavailable on Ubuntu 14.04 (libcurl 7.36)
-        //TODO: Introduce something like an #if
-        //easyHandle.set(waitForPipeliningAndMultiplexing: true)
-        //easyHandle.set(streamWeight: priority)
+	//TODO: The CURLOPT_PIPEDWAIT option is unavailable on Ubuntu 14.04 (libcurl 7.36)
+	//TODO: Introduce something like an #if, if we want to set them here
```

Comment:
May make sense to document what the CURLOPT_PIPEWAIT option helps achieve here ..